.github/workflows/bats.yaml

```diff
@@ -0,0 +1,17 @@
+name: Bats tests
+on: [push, pull_request]
+env:
+  functional_test: 'true'
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install Bats
+        run: |
+          apt-get update
+          apt-get install --assume-yes bats
```

Comment:
Note: This will install [the same version](https://packages.ubuntu.com/source/focal/bats) as the previous Travis CI configuration. I think installing from APT is easier than clone-and-make. By the way, I chose `--assume-yes` over `-y` for the sake of readability (codes are written once, read many times).